### ISSUE TITLE
scaling bug in 5.3.1 and above

### ISSUE THREAD

**Issue by rizen** [pixijs/pixi.js#7064]
### Expected Behavior

Scale the stage proportionally with the render view area. This works fine in 5.3.0 or below.

### Current Behavior

As of 5.3.1 (or higher) the content is pushed up (in the Y direction) and if it's scaled far enough it scales completely out of frame.

### Possible Solution

No idea. I don't know the internals of PIXI well enough to speculate.

### Steps to Reproduce

This is 5.3.0 aka working = https://jsfiddle.net/v6zpbc0q/  (see the white box in the upper left corner)
This is 5.3.3 aka broken = https://jsfiddle.net/krLb1gv7/ (see the white box positioned partially off screen)

### Environment

- **`pixi.js` version**: 5.3.1
- **Browser & Version**: Version 14.0 (15610.1.28.1.9, 15610) 
- **OS & Version**: Mac OS 10.15.7
- **Running Example**: https://jsfiddle.net/krLb1gv7/

I've also tried this on Chrome and Firefox and it has the same behavior there.

**Comment by rizen** [comment]
Here's the code in case the jsfiddles go away for some reason:

```
const canvas = { width : 300, height : 300 };

var app = new PIXI.Application({
    width: canvas.width,
    height: canvas.height,
    backgroundColor: 0x2c3e50
});
document.body.appendChild(app.view);

const shape = new PIXI.Graphics();
shape.beginFill(0xffffff,1);
shape.drawRect(20,20,70,100);
shape.endFill();
app.stage.addChild(shape);

// scaling
const scale_factor = 0.8;
app.stage.scale.x = scale_factor;
app.stage.scale.y = scale_factor;
app.renderer.view.width = canvas.width * scale_factor;
app.renderer.view.height = canvas.height * scale_factor;

```

**Comment by bigtimebuddy** [comment]
I swear this was fixed, @SukantPal seems like a you question.

**Comment by SukantPal** [comment]
Resizing the renderer does work correctly:

https://jsfiddle.net/ShukantPal/3o0x6q1w/

**Comment by SukantPal** [comment]
It is pushed up because the `screen.height * renderer.resolution` value doesn't equal `view.height` (which is assumed to be correct at all times). See the following:

https://github.com/pixijs/pixi.js/blob/fe7b2191e0311b8174a012366ff21c8e2b6dc153/packages/core/src/renderTexture/RenderTextureSystem.ts#L182

`viewportFrame.y` is y-inverted, by subtracting it from `view.height`. You would expect it to be zero in this case; however, `viewportFrame.height = screen.height * resolution > renderer.view.height`; that is why it is pushed up.

---

TLDR; This is not bug, but rather undefined behavior. Maybe we need to add it to our docs that resizing the view without the renderer is illegal.

**Comment by rizen** [comment]
Thank you for telling me the right way to do this. I followed a tutorial I found online, which was obviously flawed.
